translator/reader/read_light.cpp

```diff
@@ -127,7 +127,16 @@ void UsdArnoldReadDomeLight::read(const UsdPrim &prim, UsdArnoldReaderContext &c
             AiNodeSetStr(node, "format", AtString("angular"));
         }
     }
-    AiNodeSetFlt(node, "camera", 0.f);
+
+    // Special case, the attribute "color" can be linked to some shader
```

Comment:
Could you please move this code to a utility function?

Reply:
done in latest commit